test/api/templates/projects/instance-upgrade-shared-codes/001-use-ok/request.json

```diff
@@ -1,5 +1,5 @@
 {
-  "path": "/v1/repositories/keboola/templates/tmpl/default/use",
+  "path": "/v1/repositories/keboola/templates/tmpl/0.0.1/use",
```

Comment:
Na zaver doplnenie testov.

### [Template use shared codes](https://github.com/keboola/keboola-as-code/tree/michaljurecko-psgo-45-fix-shared-codes-in-tmpl/test/api/templates/repositories/template-use-shared-codes)

![image](https://user-images.githubusercontent.com/19371734/217814341-ab297802-c53b-412c-80e0-e29eeaf48a47.png)

2x sa pouzije rovnaka sablona, no vysledne existuje iba jeden shared code, ma nastavenu hodnotu z toho 2-ho pouzitia. 

Realne v produkcii Monika nebude menit obsah shared code, radsej vytvori novy, napr. s template ID `example.v2`. [Vid diskusia](https://keboola.slack.com/archives/C02C3GD6Q64/p1675851998602729?thread_ts=1675844861.724379&cid=C02C3GD6Q64).

https://github.com/keboola/keboola-as-code/blob/51c9d79cd6d487dc432c4d731bc30662f2862755/test/api/templates/repositories/template-use-shared-codes/expected-state.json#L13-L38

-------------------

### [Template upgrade shared codes](https://github.com/keboola/keboola-as-code/tree/michaljurecko-psgo-45-fix-shared-codes-in-tmpl/test/api/templates/projects/instance-upgrade-shared-codes)

![image](https://user-images.githubusercontent.com/19371734/217815013-edca8fa7-e542-4c93-aab2-86f59ad1a0f1.png)

V tomt teste sa najpr vykona `use` a potom `upgrade`, znova, vysledne existuje iba jeden shared code.

https://github.com/keboola/keboola-as-code/blob/51c9d79cd6d487dc432c4d731bc30662f2862755/test/api/templates/projects/instance-upgrade-shared-codes/expected-state.json#L13-L38